r/content/arrays.Rmd

```diff
@@ -185,4 +185,4 @@ Scalar, Array, and ChunkedArray objects.  The returned object will be an Arrow o
 
 ### See also
 
-For a more in-depth discussion of Arrow compute functions, see the section on (using arrow functions in dplyr verbs in arrow)[#Using-arrow-functions-in-dplyr-verbs-in-arrow]
+For a more in-depth discussion of Arrow compute functions, see the section on [using arrow functions in dplyr verbs in arrow](#Use-arrow-functions-in-dplyr-verbs-in-arrow)
```

Comment:
```suggestion
For a more in-depth discussion of Arrow compute functions, see the section on [using arrow functions in dplyr verbs in arrow](#use-arrow-functions-in-dplyr-verbs-in-arrow)
```

Reply:
I built it locally and it seems the link is still broken - the autogenerated section reference is case-sensitive.

Reply:
Thanks for checking that!